lib/fetch/headers.js

```diff
@@ -383,18 +384,68 @@ class Headers {
     return this[kHeadersList].set(name, value)
   }
 
+  // https://fetch.spec.whatwg.org/#dom-headers-getsetcookie
+  getSetCookie () {
+    webidl.brandCheck(this, Headers)
+
+    // 1. If this’s header list does not contain `Set-Cookie`, then return « ».
+    // 2. Return the values of all headers in this’s header list whose name is
+    //    a byte-case-insensitive match for `Set-Cookie`, in order.
+
+    return this[kHeadersList].cookies ?? []
+  }
+
+  // https://fetch.spec.whatwg.org/#concept-header-list-sort-and-combine
   get [kHeadersSortedMap] () {
```

Comment:
A Map can't be used anymore because there can now be duplicate set-cookie keys. I could probably work around it but it wouldn't be maintainable. So I chose the spec solution instead, which is probably slower, but much easier to fix issues with in the future.